extension/locale/sv-SE/vimfx.properties

```diff
@@ -0,0 +1,85 @@
+button_tooltip_enabled=VimFx är aktiverat. Klicka för att avaktivera (Shift-Alt-v)
+button_tooltip_disabled=VimFx är avaktiverat. Klicka för att aktivera (Shift-Alt-v)
+button_tooltip_blacklisted=VimFx är svartlistad på denna sidan
+button_tooltip_insertMode=VimFx är i inmatningsläge på denna sidan: Alla kommandon kommer ignoreras. Klicka för att gå ur (Esc [som standard])
+item_preferences=Inställningar
+item_blacklist_button_tooltip=Svartlista
+item_blacklist_button_inverse_tooltip=Ta bort denna regeln
+
+help_section_urls=Hantera URL:er
+help_command_focus=Fokusera adressfältet
+help_command_focus_search=Fokusera sökfältet
+help_command_paste=Gå till adressen i urklipp
+help_command_paste_tab=Öppna en ny flik och gå till adressen i urklipp
+help_command_marker_yank=Kopiera länkadressen eller inmatad text till urklipp
+help_command_marker_focus=Fokusera element
+help_command_yank=Kopiera länk till den nuvarande sidan till urklipp
+help_command_reload=Ladda om den nuvarande sidan
+help_command_reload_force=Ladda om den nuvarande sidan och alla resurser (js, css, img)
+help_command_reload_all=Ladda om sidor i alla flikar
+help_command_reload_all_force=Ladda om sidor i alla flikar, inklusive alla resurser (js, css, img)
+help_command_stop=Stoppa inladdning av den nuvarande sidan
+help_command_stop_all=Stoppa inladdning av sidor i alla flikar
+
+help_section_nav=Navigera sidan
```

Comment:
Alright, let’s go with “Navigera sidan”.

Yes it feels a bit silly to speak English with each other about Swedish, but that’s the way things are done here.